PXWeb/Resources/Styles/scss/pxweb/_PxWeb.scss

```diff
@@ -148,6 +148,28 @@
   padding-left: 1px;
 }
 
+
+
+#pxwebcontent .px-external-search-div {
+    margin: 20px;
+    margin-right: 30px;
```

Comment:
The two others have #header, is it left out on purpose here?

Reply:
Actually it is a copy of the style for px-change-lang-div. Maybe we should add header to both of them (px-external-search-div and px-change-lang-div)?

Reply:
Can't see why not :-)